pkg/metrics/internal_user_log.proto

```diff
@@ -0,0 +1,114 @@
+/*
+ * Copyright (C) 2023 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+ 
+ syntax = "proto2";
+ package metrics;
+
+ message Duration {
+  required int64 seconds = 1;
+  required int32 nanos = 2;
+}
+
+
+enum UserType {
```

Comment:
Include an Unspecified Value: i.e `UNKNOWN_USER_TYPE = 0` https://protobuf.dev/programming-guides/dos-donts/#unspecified-enum

Reply:
This proto is defined by Atest team , I didn't change it for compatibility reason .
reference is sent in chat for privacy